dpgen/generator/run.py

```diff
@@ -3662,6 +3661,8 @@ def run_iter (param_file, machine_file) :
        with open (machine_file, 'r') as fp:
            mdata = json.load (fp)
 
+    if jdata['mass_map'] == 'auto': jdata['mass_map'] = [get_atomic_masses(i) for i in jdata['type_map']]
```

Comment:
Please check if it is appropriate to modify jdata here.

Reply:
Please do not merge two lines into one line.

Reply:
I think what @wanghan-iapcm suggests is to make a new method named `update_mass_map`.

Reply:
Also, a unit test can be added for `update_mass_map`.

Reply:
How to add a unit test?

Reply:
you may check https://github.com/deepmodeling/dpgen/tree/master/tests/generator/test*py for examples